docs/integrations/llms/proxy_server.md

```diff
@@ -0,0 +1,26 @@
+## Use Anthropic,Huggingface,Palm,Ollama, etc.[Full List](https://docs.litellm.ai/docs/providers)
+
+### Create OpenAI-proxy
+We'll use [LiteLLM](https://docs.litellm.ai/docs/) to create an OpenAI-compatible endpoint, that translates OpenAI calls to any of the [supported providers](https://docs.litellm.ai/docs/providers).
+
+
+Example to use a local CodeLLama model from Ollama.ai with PromptFlow: 
+
+Let's spin up a proxy server to route any OpenAI call from PromptFlow to Ollama/CodeLlama
+
+```python
+pip install litellm
+```
+```python
+$ litellm --model ollama/codellama
+
+#INFO: Ollama running on http://0.0.0.0:8000
+```
+
+[Docs](https://docs.litellm.ai/docs/proxy_server)
```

Comment:
As the author of this documentation, the community may look to you as the expert on this LiteLLM integration. Please consider adding your support contact or linking to the LiteLLM issue tracker so users have a channel for feedback.

For example, you could append something like:

"For issues or questions on the LiteLLM integration, please create an issue on the [LiteLLM GitHub repo](https://github.com/BerriAI/litellm/issues)."

Providing a way to contact you ensures users can get help if they run into any problems following this guide. Since you developed this integration, directing questions your way would be appreciated.